app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,59 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
Better not to add styles to the manifest file - make another file alongside it - it'll be auto-loaded by require_tree